GitFyle.Core.Api.Tests.Unit/Services/Foundations/Sources/SourceServiceTests.Exceptions.RemoveById.cs

```diff
@@ -67,7 +67,7 @@ await Assert.ThrowsAsync<SourceDependencyException>(
         }
 
         [Fact]
-        private async Task ShouldThrowDependencyValidationExceptionOnRemoveByIdIfDbConcurrencyOccursAndLogItAsync()
+        public async Task ShouldThrowDependencyValidationExceptionOnRemoveByIdIfDbConcurrencyOccursAndLogItAsync()
```

Comment:
var lockedSourceException =
                new LockedSourceException(
                    message: "Locked source record error occurred, please try again.",
                    innerException: dbUpdateConcurrencyException,
                    data: dbUpdateConcurrencyException.Data);

Reply:
Resolved. Thank you.